tools/mkhtml.py

```diff
@@ -76,8 +76,8 @@ def decode(bytes_):
 src_file = "%s.html" % pgm
 tmp_file = "%s.tmp.html" % pgm
 
-trunk_url = "https://github.com/OSGeo/grass/tree/master/"
-addons_url = "https://github.com/OSGeo/grass-addons/tree/master/"
+trunk_url = "https://github.com/OSGeo/grass/tree/releasebranch_7_8/"
```

Comment:
Looks good but could the version be anyhow retrieved? (like further down in this file:)

https://github.com/OSGeo/grass/blob/02ed421865a3710acff9c99e55e99129cbc41fc2/tools/mkhtml.py#L394